src/events/messages/messageDeleteBulkRemoveResponses.ts

```diff
@@ -0,0 +1,15 @@
+import { Events } from '#lib/types/Enums';
+import { ApplyOptions } from '@skyra/decorators';
+import { Collection, Message, Snowflake } from 'discord.js';
+import { Event, EventOptions } from 'klasa';
+
+@ApplyOptions<EventOptions>({ event: Events.MessageDeleteBulk })
+export default class extends Event {
+	public async run(messages: Collection<Snowflake, Message>) {
+		for (const message of messages.values()) {
+			for (const response of message.responses) {
```

Comment:
I think it's better to emit messageDelete here we we follow DRY pattern

Reply:
Do you know how many things you break from doing this?

Reply:
Wdym? Just replace this inner for loop with `this.client.emit('messageDelete', message)`

Reply:
`messageDelete` is a discord.js event.